sale_purchase_stock_line_note/models/stock_move_line.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
```

Comment:
Can you add an header please?

Reply:
Done, let me know if there is anything else that is missing :)